README.md

```diff
@@ -1,70 +1,226 @@
-# Getting Started with Create React App
+<a name="readme-top"></a>
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+<div align="center">
 
-## Available Scripts
+  <img src="logo.png" alt="logo" width="400"  height="auto" />
```

Comment:
## [OPTIONAL]
- [ ] I think solve this by putting a valide logo or remove ut if you don’t have one this will make your project readme more  Professional